src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Rejection.scala

```diff
@@ -137,7 +137,7 @@ object Rejection {
     * @param schema a reference to the schema
     * @param report the validation report
     */
-  final case class InvalidResource(schema: Ref, report: ValidationReport)
+  final case class InvalidResource(schema: Ref, report: Json)
```

Comment:
could we preserve the validation report and handle the json conversion at the routes level?